inc/agent.class.php

```diff
@@ -1577,6 +1584,7 @@ public function getOwner() {
    /**
     * Determine the enrollment method depending on input data
     * @param array $input
+    * @return int
```

Comment:
please use integer as you choos if elsewhere already

Reply:
@DIOHz0r Remember to participate in the discuss of the Pull Request 🤓